firebase-auth.html

```diff
@@ -136,6 +136,20 @@
         },
 
         /**
+         * Authenticates a Firebase client using an oauth id_token.
+         *
+         * @return {Promise} Promise that handles success and failure.
+         */
+        signInWithCredential: function(credential) {
+          if (!this.auth) {
+            return Promise.reject('No app configured for firebase-auth!');
+          }
+
+          return this._handleSignIn(this.auth.signInWithCredential(credential));
+        },
+
+
+        /**
```

Comment:
Awesome 👍 . Now I would remove that extra new line at line 151, you see how every method in this file is separated by one new line? That should be everywhere. It'll keep the code clean and nice to maintain.